content/microcontrollers/esp32.md

```diff
@@ -26,14 +26,16 @@ The [Espressif ESP32](https://www.espressif.com/en/products/socs/esp32) is a pow
 
 ### CLI Flashing on Linux
 
-You need to install the Espressif toolchain for Linux to use TinyGo with the ESP32: 
+You need to install the Espressif toolchain for Linux to use TinyGo with the ESP32:
 
 https://docs.espressif.com/projects/esp-idf/en/release-v3.0/get-started/linux-setup.html#standard-setup-of-toolchain-for-linux
 
-In addition, you must install the `esptool` flashing tool:
+In addition, you must install the `esptool` flashing tool and clone the tinygo repo where you'll find the examples folder:
```

Comment:
No, the examples folder is already included in TinyGo (it is part of the standard library). You don't need to download the TinyGo source code for it.